extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -414,49 +293,206 @@ The valid events are as follows.
+  src: https://vendor.example/amp-video-iframe
+      ?videoid=MY_VIDEO_ID
+      &channelid=MY_CHANNEL_ID
+
+  poster: https://vendor.example/poster.jpg
+      ?videoid=MY_VIDEO_ID
+      &channelid=MY_CHANNEL_ID
+-->
+<amp-video-iframe
+  layout="responsive"
+  width="16"
+  height="9"
+  src="https://vendor.example/amp-video-iframe"
+  poster="https://vendor.example/poster.jpg"
+  data-param-videoid="MY_VIDEO_ID"
+  data-param-channelid="MY_CHANNEL_ID"
+>
+</amp-video-iframe>
+```
+
+For most video providers, <amp-video-iframe> provides enough tools for common playback actions. See methods and events to confirm. Refer to the  vendor player spec to confirm amp-video-iframe fits your needs. 
+
+### Disallowed usage of `<amp-video-iframe>`
+
+Use `<amp-video-iframe>` to embed a video player. Do not use it to embed a video directly or for advertisement purposes. 
+
+#### Embed a video directly 
+
+Use `<amp-video>` to directly embed a video on the AMP document.
+
+#### Advertising 
+
+Use `<amp-ad>` for advertising purposes. `<amp-video-iframe>` may display videos where part of the videos is advertising.
+
+Do not use `<amp-video-iframe>` for the _primary purpose_ of displaying advertising.  This component is not rendered when in violation of this policy. The reasons for this policy:
+
+- `<amp-video-iframe> `enforces sandboxing to itself and child iframes. This means landing pages may break, even if the ad itself appears to work.
+
+- `<amp-video-iframe>` has no controlled resize mechanism.
+
+
+
+## Attributes
+
+### `src`
+
+Point the `src` attribute to the video.
+
+The `src` attribute behaves like on a standard iframe with one exception. The `<amp-video-iframe>` component appends the `#amp=1` fragment the URL. This tells documents they are embedded within the AMP context. This fragment is only added if the URL specified by `src` does not already have a fragment. 
+
+### `poster`
+
+Points to an image URL. The `<amp-video-iframe>` component displays this image while the video loads.
+
+### `autoplay` (optional)
+
+Adding this attribute plays the video automatically, in supported browsers, when it is visible to the user. You must meet the conditions outlined in the [Video in AMP spec](../..//spec/amp-video-interface.md#autoplay). 
+
+### `dock` (optional)
+
+Adding this attribute minimizes and fixes a manually played video to the corner of the screen, or specified element, as the user scrolls. 
+
+This behavior requires dual use of the [`<amp-video-docking>`](../amp-video-docking/amp-video-docking.md) component. 
+
+### `implements-media-session` (optional)
+
+Include the `implements-media-session` attribute if the document inside the iframe implements the [MediaSession API](https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API) independently.
+
+### `implements-rotate-to-fullscreen` (optional)
+
+Include the `mplements-rotate-to-fullscreen` attribute if the document inside the iframe implements rotate-to-fullscreen independently.
```

Comment:
```suggestion
Include the `implements-rotate-to-fullscreen` attribute if the document inside the iframe implements rotate-to-fullscreen independently.
```

We also need to mention the [`rotate-to-fullscreen` attribute](https://amp.dev/documentation/components/amp-video/#attributes) itself.